tests/all/integration/auth_helper.py

```diff
@@ -14,7 +14,7 @@ def create_user(email, password, is_verified=True):
 
 
 def create_super_admin(email, password):
-    user = create_user(email, password, is_verified=True)
```

Comment:
so what does this accomplish, Super Admin is a user, why can't it be verified by default?

Reply:
Also how is this related to the issue it solves. I'm confused, can you please elaborate.

Reply:
@CosmicCoder96 the `create_user` function has the `is_verified` argument to be true by default so passing it as a true for a super admin is more or less redundant.

As for how it solves the issue, kindly bear with me because it was more of an observation task :
I could not create a `verified` super-user on using `create_db.py` upon db initialisation but the subsequent super users being created by the same place are all verified as they should be.

On removing this `is_verified` argument from the function call, the first super admin being created is verified.

This weird behaviour is being shown in my system for quite a time now but it would be great if someone could verify it too